rust/src/transformations/quantile_score_candidates/check_candidates.tex

```diff
@@ -0,0 +1,48 @@
+\documentclass{article}
+\input{../../lib.sty}
+
+\title{\texttt{fn check\_candidates}}
+\author{Michael Shoemate}
+\begin{document}
+\maketitle  
+
+
+\contrib
+
+Proves soundness of \rustdoc{transformations/quantile\_score\_candidates/fn}{check\_candidates} 
+in \asOfCommit{mod.rs}{f5bb719}.
+\texttt{check\_candidates} raises an error if the discrete quantile candidate set is invalid.
+
+\section{Hoare Triple}
+\subsection*{Precondition}
+None
+
+\subsection*{Function}
+\label{sec:python-pseudocode}
+\lstinputlisting[language=Python,firstline=2,escapechar=`]{./pseudocode/check_candidates.py}
+
+\subsection*{Postcondition}
+
+\begin{theorem}
+    Candidates must be:
+    \begin{enumerate}
+        \item non-empty
+        \item strictly increasing
+        \item totally ordered
+    \end{enumerate}
+
+    Otherwise the function errors.
+\end{theorem}
+
+\begin{proof}
+    \begin{enumerate}
+        \item \texttt{candidates} is non-empty, by the check on line \ref{empty}
```

Comment:
This renders as 
<img width="816" alt="Screenshot 2025-04-17 at 11 13 37 AM" src="https://github.com/user-attachments/assets/7aa9dd50-869b-4c1f-88de-a5d8d2ed1f29" />

Maybe it expects some text between the `\begin{proof}` and `\begin{enumerate}`?